lms/templates/student_account/login.underscore

```diff
@@ -1,14 +1,24 @@
 <div class="js-form-feedback" aria-live="assertive" tabindex="-1">
 </div>
 
-<% if ( context.createAccountOption !== false && !context.syncLearnerProfileData) { %>
+<% if ( context.createAccountOption !== false && context.enterpriseName == "") { %>
```

Comment:
why did you remove `!context.syncLearnerProfileData` ? I think this is an independent check and should stay in here. You can add `context.enterpriseName == "" ` after it with an `&&`.

Reply:
ah, it is deleted mistakenly.